### ISSUE TITLE
Hi Roy!

### ISSUE THREAD

**Issue by arthurvanl** [RoyWiersma/Minty#1]
Kan ik je ergens mee helpen?

**Comment by RoyWiersma** [comment]
Ja, ik had namelijk gesolliciteerd op de front end stage met React en de opdracht is back end met laravel, waar ik nooit mee heb gewerkt. Dus mijn vraag was of de opdracht zo kan worden gemaakt dat ik React front end erin kan doen

**Comment by arthurvanl** [comment]
Het is zelfs de bedoeling dat je het in react/nextjs (stond in de mail). De laravel zit er alleen bij zodat wij zelf ermee konden testen met de opracht die wij hadden bedacht.

**Comment by RoyWiersma** [comment]
Oh zo, ja het schept wat verwarring van wat nou precies de bedoeling is

**Comment by arthurvanl** [comment]
Het is niet in een bestand nee. Je moet die [cors](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) handling oplossen

**Comment by arthurvanl** [comment]
Ik zie wel een dingetje. Je maakt een get call, maar het moet een post call zijn.

Daarnaast is de nieuwe api url dit geworden: https://dev.api.mintycloud.nl/api/v2.1

**Comment by RoyWiersma** [comment]
Waarom moet ik een post call doen als ik data wil ophalen? Ik heb mijn code aangepast en ik krijg nu een status 404

**Comment by arthurvanl** [comment]
De endpoint is er voor om te kijken of er bepaalde domeinen te koop staan of al geregistreerd staan. Om die domeinen te checken moet je eerst data versturen (request body). Met deze reden hebben wij ervoor gekozen om er een POST request te maken.

Overigens staat er in de opdracht dat je een POST request moet maken:
![image](https://github.com/RoyWiersma/Minty/assets/20114845/40a7dc5a-0ffd-4478-bef6-3950076f6429)

**Comment by RoyWiersma** [comment]
Ja dat zag ik het is alleen een beetje vreemd om data te krijgen met een post ipv een get

**Comment by RoyWiersma** [comment]
Oke ik krijg nu een status 401 unauthorized.
Dus ik ga ervan uit dat dit niet helemaal correct is
![image](https://github.com/RoyWiersma/Minty/assets/36405434/59cb683f-cb19-4e09-99b3-bc00e96f716a)

**Comment by arthurvanl** [comment]
Klopt api key is verkeerd:
072dee999ac1a7931c205814c97cb1f4d1261559c0f6cd15f2a7b27701954b8d

**Comment by arthurvanl** [comment]
> Ja dat zag ik het is alleen een beetje vreemd om data te krijgen met een post ipv een get

is niet gek als je ook data gaat versturen

**Comment by RoyWiersma** [comment]
Dus dit zou nu de correcte code zijn: 
![image](https://github.com/RoyWiersma/Minty/assets/36405434/b4e2db65-d02b-4be2-8902-6059cbd35f14)

**Comment by arthurvanl** [comment]
Nee

geen object maar een directe array

**Comment by RoyWiersma** [comment]
![image](https://github.com/RoyWiersma/Minty/assets/36405434/02a7ad0f-a30b-40d0-9460-2c99bda3a1e1)
Oke nu krijg ik een error 501

**Comment by arthurvanl** [comment]
Het is /domains/search

**Comment by RoyWiersma** [comment]
Oke ik ben klaar met de opdracht.

**Comment by arthurvanl** [comment]
Hi Roy, zou je de repository op private kunnen zetten? Je zal binnenkort eens een mail krijgen wanneer je op gesprek kan komen!

**Comment by RoyWiersma** [comment]
Oke prima
